kolibri/utils/options.py

```diff
@@ -304,135 +257,207 @@ def origin_or_port(value):
         },
         "CHERRYPY_THREAD_POOL": {
             "type": "integer",
             "default": calculate_thread_pool(),
-            "envvars": ("KOLIBRI_CHERRYPY_THREAD_POOL",),
+            "description": "How many threads the Kolibri server should use to serve requests",
         },
         "CHERRYPY_SOCKET_TIMEOUT": {
             "type": "integer",
             "default": 10,
-            "envvars": ("KOLIBRI_CHERRYPY_SOCKET_TIMEOUT",),
+            "description": """
+                How long a socket should wait for data flow to resume before
+                it considers that the connection has been interrupted.
+                Increasing this may help in situations where there is high latency on a network,
+                or the bandwidth is bursty, and it is expected that data flow may be interrupted
+                but not indicative of the connection failing.
+            """,
         },
         "CHERRYPY_QUEUE_SIZE": {
             "type": "integer",
             "default": 30,
-            "envvars": ("KOLIBRI_CHERRYPY_QUEUE_SIZE",),
+            "description": """
+                How many requests to allow in the queue.
+                Increasing this may help situations where requests are instantly refused by the server.
+            """,
         },
         "CHERRYPY_QUEUE_TIMEOUT": {
             "type": "float",
             "default": 0.1,
-            "envvars": ("KOLIBRI_CHERRYPY_QUEUE_TIMEOUT",),
+            "description": """
+                How many seconds to wait for a request to be put into the queue.
+                Increasing this may help situations where requests are instantly refused by the server.
+            """,
         },
         "PROFILE": {
             "type": "boolean",
             "default": False,
             "envvars": ("KOLIBRI_SERVER_PROFILE",),
+            "description": "Activate the server profiling middleware.",
+        },
+        "DEBUG": {
+            "type": "boolean",
+            "default": False,
+            "description": "Run Kolibri with Django setting DEBUG = True",
         },
-        "DEBUG": {"type": "boolean", "default": False, "envvars": ("KOLIBRI_DEBUG",)},
         "DEBUG_LOG_DATABASE": {
             "type": "boolean",
             "default": False,
-            "envvars": ("KOLIBRI_DEBUG_LOG_DATABASE",),
+            "description": "Activate debug logging for Django ORM operations.",
         },
     },
     "Paths": {
         "CONTENT_DIR": {
-            "type": "string",
+            "type": "path",
             "default": "content",
-            "envvars": ("KOLIBRI_CONTENT_DIR",),
+            "description": """
+                The directory that will store content files and content database files.
+                To change this in a currently active server it is recommended to use the
+                content movedirectory management command.
```

Comment:
Put the command between single quotes:
`'movedirectory' content management command.`

Reply:
for reasons that escape me, the command is actually `content movedirectory`

Reply:
I have obviously not tested that command in a while.... 🤦🏽‍♀️